content/blog/2018/2018-09-01-cccc-rounds-1-2-and-3.md

```diff
@@ -213,7 +215,7 @@ Games can be seen from the following site:
 
 So far we have the results:
 
-**ROUND 1**  
+&emsp;&emsp;**ROUND 1**  
```

Comment:
Btw what I usually did in those cases is converted them to `##` or `###` headers. As logically they are headers.